packages/asset-swapper/src/utils/rfq_client_mappers.ts

```diff
@@ -0,0 +1,23 @@
+import { FillQuoteTransformerOrderType } from '@0x/protocol-utils';
```

Comment:
consider just putting this in the interface file -- will be easier to clean up later